internal/artifacts/nighttimewhispersintheechoingwoods/config.yml

```diff
@@ -0,0 +1,2 @@
+key: "nighttimewhispersintheechoingwoods"
+text_map_id: 994089547
```

Comment:
change to 279470883 and rerun the pipeline
for next time: use the entry in the dm json that has "openConfig"